tools/kagami/src/genesis.rs

```diff
@@ -213,33 +213,38 @@ fn generate_synthetic(
     accounts_per_domain: u64,
     assets_per_domain: u64,
 ) -> color_eyre::Result<RawGenesisBlock> {
-    // Add default `Domain` and `Account` to still be able to query
-    let mut builder = RawGenesisBlockBuilder::default()
-        .domain("wonderland".parse()?)
-        .account("alice".parse()?, crate::DEFAULT_PUBLIC_KEY.parse()?)
-        .finish_domain();
+    let mut genesis = generate_default(executor)?;
```

Comment:
i needed this change because i wanted to generate large genesis while still being able to run tps benchmark (which is built with default genesis in mind)